generators/Exercises/Generators/GoCounting.cs

```diff
@@ -72,7 +72,7 @@ private string RenderTerritoryAssert()
         private string RenderTerritoriesAssert()
         {
             var territoriesAssert = new StringBuilder();
-            territoriesAssert.AppendLine(Render.AssertEqual("expected.Keys", "actual.Keys"));
+            territoriesAssert.AppendLine(Render.AssertEqual("expected.Keys.Count", "actual.Keys.Count"));
```

Comment:
being that al the keys are used in the assertions below checking the count is sufficient.
alternative approaches would be to convert the key collections to sets or to just remove this assertion

Reply:
I think: just remove the assertion.

Reply:
done